### DISCUSSION TITLE
Can I use MergerFS and snapraid on extra disks?

### DISCUSSION THREAD

**Discussion by rishid** [siderolabs/talos#7741] [Q&A] [answered]
I have a compute/storage combined home server that I use [MergerFS](https://github.com/trapexit/mergerfs) and SnapRaid presently with and thinking of moving over to Talos.

Is this possible with using Talos? I can't seem to find much information on the docs about how disks are managed, formatted, partitioned using Talos. The only thing I found was a reference to [MachineDisk](https://www.talos.dev/v1.5/reference/configuration/#machinedisk)

**Comment by smira** [comment, marked as answer]
If that doesn't require Linux kernel support which isn't present in Talos, you could run everything as static pods/daemonsets. 

Talos itself doesn't touch extra disks if you don't use machine config's `MachineDisk`.

**Reply by rishid** [reply]
That is helpful, it just requires FUSE support which appears to be compiled into the Talos kernel based on Kernel config I found. I will give it a shot and see what happens.
